src/main/java/de/hpi/bpt/argos/persistence/database/DatabaseConnectionImpl.java

```diff
@@ -70,7 +72,8 @@ public boolean setup() {
 			databaseSessionFactory = configuration.buildSessionFactory();
 
 		} catch (ServiceException e) {
-			logger.error("can't connect to the database server", e);
+			logger.error("Can't connect to the database server");
+			logger.trace("Reason: ", e);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Reason: " 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1192)